queries/elixir/rainbow-delimiters.scm

```diff
@@ -20,6 +20,10 @@
   "[" @delimiter
   "]" @delimiter @sentinel) @container
 
+(access_call
+  "[" @opening
+  "]" @closing @sentinel) @container
```

Comment:
Capture names have change, we no longer differentiate between different kinds of delimiters. I am surprised the code worked for you at all.

```suggestion
  "[" @delimiter
  "]" @delimiter @sentinel) @container
```